skbase/_base.py

```diff
@@ -466,13 +466,8 @@ def create_test_instance(cls, parameter_set="default"):
         else:
             params = cls.get_test_params()
 
-        if isinstance(params, list):
```

Comment:
is this covered by the test then?

What if a user implements sth but does not test?

Reply:
I just simplified the logic in the if-else statements.

Reply:
fair enough